src/python/espressomd/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+file(GLOB cython_SRC *.pyx)
+file(GLOB cython_HEADER *.pxd)
+file(GLOB cython_AUX *.py)
+file(COPY ${cython_AUX} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+
+add_custom_command(OUTPUT gen_pxiconfig.cpp
+                   COMMAND ${PYTHON_EXECUTABLE}
+                   ${CMAKE_CURRENT_SOURCE_DIR}/gen_pxiconfig.py
+                   ${CMAKE_SOURCE_DIR}/src/features.def ${CMAKE_CURRENT_BINARY_DIR}/gen_pxiconfig.cpp
+                   DEPENDS ${CMAKE_SOURCE_DIR}/src/features.def
+                  )
+
+include_directories(${CMAKE_BINARY_DIR}/src)
+include_directories(${CMAKE_SOURCE_DIR}/src/core)
```

Comment:
Don't need `${CMAKE_SOURCE_DIR}` here, `include_directories` takes paths relative to the current source directory. So you could use `../../core`.